cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedInitialization.qll

```diff
@@ -1009,6 +1067,8 @@ class TranslatedConstructorBareInit extends TranslatedElement, TTranslatedConstr
     result = this.getParent().getChildSuccessor(this, kind)
   }
 
+  override Instruction getALastInstructionInternal() { none() } // FIXME: does this need to be filled in?
```

Comment:
Here's a FIXME that we haven't resolved yet. What's the status of this one?

Reply:
That class only exists to handle an (old?) extractor bug. I don't think it can ever have a destructor attached, and neither can the init list it's part of, so I think it's fine for `getALastChild` to have no results.

Reply:
Sounds good. Should we delete the `FIXME` comment then? And maybe convert it to an internal issue?